src/main/java/Duke.java

```diff
@@ -22,7 +22,7 @@ public static void exit() {
     }
 
     public static void main(String[] args) {
-        ArrayList<String> userTexts = new ArrayList<String>();
+        ArrayList<Task> tasks = new ArrayList<Task>();
```

Comment:
perhaps you could use a more descriptive name for your ArrayList
```suggestion
        ArrayList<Task> taskList = new ArrayList<Task>();
```